src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -83,6 +86,22 @@ class AdditionalTab extends React.PureComponent<Props, State> {
         return TabIcon;
     }
 
+    withFtuxTooltip(body: React.Node) {
+        const { ftuxTooltipData, isLoading } = this.props;
+
+        if (!ftuxTooltipData || isLoading) {
+            return body;
+        }
+
+        const { targetingApi, text } = ftuxTooltipData;
```

Comment:
Could spread this directly onto the `AdditionalTabFtuxTooltip` since the other props are the two destructured